protos/table.proto

```diff
@@ -171,6 +171,14 @@ message DataFragment {
   // the current number of rows, subtract `deletion_file.num_deleted_rows` from
   // this value.
   uint64 physical_rows = 4;
+
+  // Blob files that are referenced by the fragment.
+  //
+  // A fragment may reference multiple blob files.
+  // Multiple fragments may reference the same blob file (e.g. when two fragments
+  // are compacted into a third combined fragment both the original fragment and the
+  // combined fragment will reference the same blob files).
+  repeated BlobFile blobs = 5;
```

Comment:
Can multiple fragment reference the same blob? I would think so if we allow data to be re-clustered. I wonder if they should be listed separate.

I also wonder how many of these we could eventually end up with. I guess not more than the number of fragments in most cases.